### DISCUSSION TITLE
Prose Color with daisy UI might have some issue

### DISCUSSION THREAD

**Discussion by theetherGit** [saadeghi/daisyui#1563] [Q&A] [answered]
@saadeghi Hello, Great fan of your work.
I was working on a small project where i need to use Multiple themes and Prose class but when i'm using prose, in some themes content is not readable. Can you help me where i'm wrong or is there any issue.

Source Code: https://github.com/etherCorps/sveltekit-devto-blog   // Sveltekit based project
Demo: https://sveltekit-devto-blog.vercel.app    // Click on any blog and try Dark, SynthWave, content is not readable.

Please help me out in this one.

**Comment by saadeghi** [comment, marked as answer]
Tailwind's typography plugin applies their custom color choices to the text but when your background is using a daisyUI color (which changes based on the current theme) those typography colors wouldn't look good on all themes (because they don't change based on the current theme)

[There's a style in daisyUI](https://github.com/saadeghi/daisyui/blob/master/src/components/styled/typography.css) to fix all that but since you sat [`styled` config to `false` here](https://github.com/etherCorps/sveltekit-devto-blog/blob/c5568fd11b1be23e570f01363a96ff72a009e2a3/tailwind.config.cjs#L29) it's not included in your project. You can manually add [that style](https://github.com/saadeghi/daisyui/blob/master/src/components/styled/typography.css) to your CSS and all those colors will be readable on all themes automatically.

Let me know if you have any questions.

**Reply by theetherGit** [reply]
Thank You so much
